XSLT/knoxcountydctomods_newsets.xsl

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+    xmlns:oai_dc='http://www.openarchives.org/OAI/2.0/oai_dc/' xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:dc="http://purl.org/dc/elements/1.1/"
+    version="2.0" xmlns="http://www.loc.gov/mods/v3">
```

Comment:
You have two unused namespaces being carried over to your resulting MODS.  Add this instruction as an attribute on your xsl:stylesheet element:

```
exclude-result-prefixes="dc oai_dc"
```

Reply:
Thanks for this - I completely missed this. I've committed this change.